api/environments/views.py

```diff
@@ -69,7 +71,7 @@
     ),
 )
 class EnvironmentViewSet(viewsets.ModelViewSet):
-    lookup_field = "api_key"
+    lookup_field = "api_key"  # This is deprecated please uuid
```

Comment:
I don't really like this approach - the documentation is going to be incorrect. Shouldn't we just add a new viewset? Can the django urls routing logic tell the difference between our API key and a uuid? It should be able to, right?

Reply:
In general, I'm also not sure why we've gone with this approach vs the approach on other models of adding the `get-by-uuid` route?

Reply:
> In general, I'm also not sure why we've gone with this approach vs the approach on other models of adding the get-by-uuid route?

Because we will need this viewset as part of `allow rotation of client side environmen key `